examples/persist-delete/persist_delete.bal

```diff
@@ -0,0 +1,13 @@
+import ballerina/io;
+import rainier.store;
```

Comment:
Order of the imports.

Reply:
Fixed in [c1be323](https://github.com/ballerina-platform/ballerina-distribution/pull/4429/commits/c1be3230aa93ba7a9aaa73823a67994b0d1e85fe)